src/index.ts

```diff
@@ -174,8 +173,6 @@ class Envoy {
       "host": "0.0.0.0", // "localhost",
     });
 
-    await this.connected;
```

Comment:
This is no longer needed since we wait for the conductor to be connected each time we do a conductor call. Also, leaving this in prevents the RPC methods from being registered in a timely manner meaning that instead of a `Conductor disconnected` error in chaperone you'll get a `Method not found` error.